index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="normalize.css" />
+    <link rel="stylesheet" type="text/css"  href="styles.css" >
+    <script src="index.js" defer></script>
+</head>
+<body>
+
+<section>
```

Comment:
section and all contents should be 
1. changed for a main
2. tabbed in for formatting